doc/modules/data_ops/basics/building_data_ops_plan.rst

```diff
@@ -0,0 +1,94 @@
+.. currentmodule:: skrub
+
```

Comment:
That's a lot of code for a documentation page. I typically prefer moving these to an example, I find it easier to maintain.

Reply:
The same comment applies to many of the pages in this section.

We could make a separate, dedicated gallery, that is inserted here, if useful